### DISCUSSION TITLE
Where else can I get OpenSSL for Windows?

### DISCUSSION THREAD

**Discussion by cherinyy** [curl/curl-for-win#48] [Q&A] [answered]
Hi,

I have previously been using the OpenSSL for Windows build provided by you and it has really saved me a lot of work, thank you very much!

But it seems like you guys don't offer that build anymore, and I was wondering where else I could get it from.

Best regards,
wordlesswind

**Comment by bagder** [comment, marked as answer]
@wordlesswind that seems like a question for the OpenSSL project...

**Reply by cherinyy** [reply]
Oh...it does.

**Reply by garoto** [reply]
Use your favorite search engine and search for "openssl {w32|windows compiles|windows builds|...}" and hope for the best. I personally use a cross-compiled LibreSSL `openssl.c` as my goto CLI solution, but yeah, I too miss when `curl-for-win` would have 3rd party binaries as part of every CI run.
